Software Construction Fundamentals/Session3/StringOperations/Strings.java

```diff
@@ -0,0 +1,194 @@
+package StringOperations;
+
+import java.util.*;
+
+public class Strings {

```

Comment:
Rename the Class name to be more specific

Reply:
changed to StringOperations